scripts/generate-licenses.py

```diff
@@ -171,7 +171,7 @@ def dump(licenses, versions, descriptions, gas, out):
         info = descriptions.get(ga, ' ')
         if ga in gas:
             gas_left.remove(ga)
-            lines.append('| ' + jar + ' | ' + info + ' | ' + version + ' | ' + ', '.join(license_list) + ' |')
+            lines.append('| ' + group + ':' + artifact + ':' + version + ' | ' + jar + ' | ' + info + ' | ' + version + ' | ' + ', '.join(license_list) + ' |')
```

Comment:
This is the generation of the .md file whose goal is to end up in https://doc.nuxeo.com/nxdoc/licenses/. I'm not sure why the GAV is useful in that page, and the NXP doesn't say.

Reply:
That was the feature request. See https://jira.nuxeo.com/browse/NXP-25263?focusedCommentId=351254&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-351254

For sure the GAV is useful and filename not reliable or meaningful enough.

The way it is displayed can still be reviewed, there was no strong request on this aspect.